modules/cnv-web-console-overview-dashboard.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+// * assembly/architecture
+
+[id="web-console-overview-dashboard_{context}"]
+= Overview Dashboard
+
+* *Details* card provides an brief overview of informational cluster details, including:
+** Cluster name
+** Provider
+** Version number of {CNVProductName} Container Platform.
+* *Inventory* card alerts administrators when direct intervention is required to resolve problems.  Information includes:
+** Number of nodes
+** Number of pods
+** Persistent storage volume claims
+** Virtual machines
+** Bare metal hosts in the cluster, listed according to their state.
+
+[NOTE]
+====
+States listed on this card are "healthy," "not healthy," "attempting to self-heal," and "not healthy and unable to recover without administrator assistance."
```

Comment:
Perhaps take the terms out of quotations and embolden them instead?

Reply:
Moved the states up to the top of the card summary, didn't like them in a NOTE. 

Question for @rawagner --- Is the state called "progress" or "in progress"? Thanks.

Reply:
we just show an icon, there is no text for the status. But i guess `in progress` is better in docs

Reply:
Cool, thanks. Done.